### ISSUE TITLE
1.18.1 Fabric Server Crash on Start up

### ISSUE THREAD

**Issue by BinxetyBinx** [samolego/Taterzens#70]
Tried adding the mod to the AOF5 pack that we're running on a server. The server was running prior to this but I thought it would be a really cool addition. Unfortunately it crashed on start up. Removing the mod the server launches fine again.

1.18.1 Fabric Server

Version of mod is [1.18.1] Taterzens 1.7.1
Version of AOF5 is 0.2.7 ( I can definitely report it to them if it's not an issue on your end, but I'm not savvy enough to figure out whose fault it is. )

To Reproduce
1. Add the mod
2.  Launch the server 

Crash log

https://pastebin.com/8N01bF3M

**Comment by samolego** [comment]
```
Error loading class: io/github/apace100/apoli/power/PreventItemUsePower (java.lang.ClassNotFoundException: io/github/apace100/apoli/power/PreventItemUsePower)
```
[This mod](https://github.com/apace100/apoli) might be causing it?

**Comment by BinxetyBinx** [comment]
Is there a way I can fix anything on my end?

**Comment by samolego** [comment]
I'm still figuring it out. Is that above the whole log? As it looks kinda different from what I've expected ...

**Comment by samolego** [comment]
```
- fabric-registry-sync-v0 0.9.0+befed49e65 via fabric
- vanillaexcavators 1.1.0
- villager-hats 1.2.1
- waystones 2.4.1
- wilderworld 1.0.0
- winterly 0.3.1
- wirelessnetworks 1.4.1
- wolveswitharmor 1.7.0-1.18
- worldedit 7.2.8+6008-1246d61
- wthit 4.4.0
- xaerominimap 21.23.0
- xaeroworldmap 1.19.0
- yigd 1.2.10
```

Taterzens aren't mentioned here, this is the wrong log ;)

**Comment by BinxetyBinx** [comment]
I threw the mod back in and grabbed the latest.log which does look a bit longer

[2022-01-05-1.log](https://github.com/samolego/Taterzens/files/7817864/2022-01-05-1.log)

**Comment by BinxetyBinx** [comment]
> ```
> - fabric-registry-sync-v0 0.9.0+befed49e65 via fabric
> - vanillaexcavators 1.1.0
> - villager-hats 1.2.1
> - waystones 2.4.1
> - wilderworld 1.0.0
> - winterly 0.3.1
> - wirelessnetworks 1.4.1
> - wolveswitharmor 1.7.0-1.18
> - worldedit 7.2.8+6008-1246d61
> - wthit 4.4.0
> - xaerominimap 21.23.0
> - xaeroworldmap 1.19.0
> - yigd 1.2.10
> ```
> 
> Taterzens aren't mentioned here, this is the wrong log ;)

It absolutely was as it was the console literally right after putting that mod in. But what I originally sent is shorter than the one I sent a minute ago.

**Comment by samolego** [comment]
https://github.com/samolego/Taterzens/releases/tag/1.7.2 should fix this

**Comment by BinxetyBinx** [comment]
New issue. Added the updated file you sent me, server started! However I tried adding TraderNPCs version 1.0.3 [Fabric] and the server is crashing again. Here's the full latest.log

[latest.log](https://github.com/samolego/Taterzens/files/7817998/latest.log)

**Comment by samolego** [comment]
Ohno, seems like 1.7.3 will be needed 😅 ...

**Comment by samolego** [comment]
It's strange though as it didn't happen in dev ...

**Comment by samolego** [comment]
If you have time, could you try this buidl https://github.com/samolego/Taterzens/actions/runs/1661864431 ?
Click on artifacts -> unzip it -> try to crash again

**Comment by BinxetyBinx** [comment]
Which of these do you want me to install? 
![image](https://user-images.githubusercontent.com/47510715/148410683-af02927a-9917-4afb-bcde-c63399e42b61.png)

**Comment by BinxetyBinx** [comment]
I assumed all. Crashed again. Here is the log
[latest (1).log](https://github.com/samolego/Taterzens/files/7823050/latest.1.log)
.

**Comment by samolego** [comment]
Just the -fabric one

**Comment by samolego** [comment]
Make sure to use fapi latest beta as well

**Comment by samolego** [comment]
Should be finally fixed in 1.8.0
